package.json

```diff
@@ -1,5 +1,5 @@
 {
-  "name": "prop-types",
+  "name": "@catalytic/prop-types",
   "version": "15.6.0",
   "description": "Runtime type checking for React props and similar objects.",
   "main": "index.js",
```

Comment:
Might want to update `license` to our generic copyright.

Reply:
And `repository` and `homepage` to this repository.

Reply:
👍  good call, updated both